prisma/seed.ts

```diff
@@ -8,6 +8,36 @@ async function main() {
   let createdUsersCounter = 0;
   const existingUsersCount = await prisma.user.count();
 
+  let createdRepositoriesCounter = 0;
+  const existingRepositoriesCount = await prisma.repository.count();
+
+  if (
+    !(await prisma.repository.findUnique({ where: { name: 'StartUI Web' } }))
```

Comment:
We need to keep marketing consistency.

```suggestion
    !(await prisma.repository.findUnique({ where: { name: 'Start UI [web]' } }))
```